windows/src/ext/sentry/test/Sentry.Client.Test.SentryClientTestMain.pas

```diff
@@ -29,7 +29,7 @@ function Init: TSentryClient;
   //o.DSN := 'https://7b1ff1dae2c8495b84f90dadcf512b84@sentry.io/4853461';
   o.Release := 'keyman-14.0.22-alpha-local';
   o.HandlerPath := 'c:\Projects\keyman\app\windows\src\ext\sentry\test\Win32\Release\crashpad_handler.exe';
```

Comment:
Should this be a relative path to `ext\sentry\2.3\crashpad_handler.exe`?

Reply:
No, this is a test unit only, not required for distribution.